src/wallet/coinselection.cpp

```diff
@@ -449,19 +450,21 @@ void OutputGroupTypeMap::Push(const OutputGroup& group, OutputType type, bool in
     }
 }
 
-CAmount GetSelectionWaste(const std::set<std::shared_ptr<COutput>>& inputs, CAmount change_cost, CAmount target, bool use_effective_value)
+CAmount SelectionResult::GetSelectionWaste(CAmount change_cost, CAmount target, bool use_effective_value)
```

Comment:
we need tests for waste calculation with bumpfee and especially for bumpfee discount

Reply:
Added a test for waste calculation with bumpfee and bumpfee discount.